src/events/userChanged.js

```diff
@@ -8,7 +8,10 @@ export default async ({ event }) => {
   try {
     const { user } = event;
     const statusEmoji = user.profile.status_emoji;
-    if (statusEmoji?.includes("som-")) {
+    if (statusEmoji?.includes("som-") && 
```

Comment:
stamping but would vastly prefer such logic determinations cleanly separated out into a method with a good name - for instance "isNumericStatusEmoji" or something